node/src/accountant/payment_adjuster/logging_and_diagnostics/log_functions.rs

```diff
@@ -21,72 +19,122 @@ Passed successfully adjustment by transaction fee, then rechecked the service fe
 applied on the adjusted set, but discovered a shortage of MASQ not to suffice even for a single \
 transaction. Operation is aborting.";
 
-const BLANK_SPACE: &str = "";
+const EMPTY_STR: &str = "";
 
-pub fn format_brief_adjustment_summary(
-    original_account_balances_mapped: HashMap<Wallet, u128>,
+pub fn accounts_before_and_after_debug(
+    original_account_balances_mapped: HashMap<Address, u128>,
     adjusted_accounts: &[PayableAccount],
 ) -> String {
-    fn format_summary_for_included_accounts(
-        original_account_balances_mapped: &HashMap<Wallet, u128>,
-        adjusted_accounts: &[PayableAccount],
-    ) -> String {
-        adjusted_accounts
-            .iter()
-            .sorted_by(|account_a, account_b| {
-                Ord::cmp(&account_b.balance_wei, &account_a.balance_wei)
-            })
-            .map(|account| {
-                format!(
-                    "{} {}\n{:^length$} {}",
-                    account.wallet,
-                    original_account_balances_mapped
-                        .get(&account.wallet)
-                        .expectv("initial balance")
-                        .separate_with_commas(),
-                    BLANK_SPACE,
-                    account.balance_wei.separate_with_commas(),
-                    length = WALLET_ADDRESS_LENGTH
-                )
-            })
-            .join("\n")
-    }
-    fn format_summary_for_excluded_accounts(excluded: &[(&Wallet, u128)]) -> String {
-        let title = once(format!(
-            "\n{:<length$} Original\n",
-            "Ruled Out Accounts",
-            length = WALLET_ADDRESS_LENGTH
-        ));
-        let list = excluded
-            .iter()
-            .sorted_by(|(_, balance_account_a), (_, balance_account_b)| {
-                Ord::cmp(&balance_account_b, &balance_account_a)
-            })
-            .map(|(wallet, original_balance)| {
-                format!("{} {}", wallet, original_balance.separate_with_commas())
-            });
-        title.chain(list).join("\n")
-    }
+    let excluded_wallets_and_balances =
+        preprocess_excluded_accounts(&original_account_balances_mapped, adjusted_accounts);
+    let excluded_accounts_summary = excluded_wallets_and_balances.is_empty().not().then(|| {
+        write_title_and_summary(
+            &excluded_accounts_title(),
+            &format_summary_for_excluded_accounts(&excluded_wallets_and_balances),
+        )
+    });
+    let included_accounts = write_title_and_summary(
```

Comment:
You may add `_summary` to this variable, so that the variable above matches with this one.